apiserver/src/api/mod.rs

```diff
@@ -111,7 +111,7 @@ pub struct APIServerSettings<T: BottlerocketShadowClient> {
 pub async fn run_server<T: 'static + BottlerocketShadowClient>(
     settings: APIServerSettings<T>,
     k8s_client: kube::Client,
-    prometheus_exporter: opentelemetry_prometheus::PrometheusExporter,
+    prometheus_registry: prometheus::Registry,
```

Comment:
Regarding the named meter, have we considered pass in the `SdkMeterProvider` as a parameter here. And create a meter in the function like 

```
let meter = provider.meter("apiserver");

// Set up metrics request builder
let request_metrics = RequestMetricsBuilder::new().build(apiserver_meter);
```

Reply:
Alternative would be to just directly call `global::meter_provider` and get the GlobalMeterProvider for the application and create the meter by using this global provider. We called `global::set_meter_provider` [here](https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/602/files#diff-028a1486656b9d1f929ac3420c6345e2527f3390725dbeff222c2f2e200ddf2dR48).

Referring to their official [doc](https://docs.rs/opentelemetry/latest/opentelemetry/global/fn.meter_provider.html).